conda/meta.yaml

```diff
@@ -20,6 +20,7 @@ build:
 test:
   requires:
     - ts-conda-build =0.3
+    - ts-utils
```

Comment:
ts-utils is a dependency of salobj already, so probably ok to not have it here.

Reply:
Personally I prefer to list all packages that the package *directly* uses. But I will certainly omit it if you prefer. If you want me to omit it here then should I also omit it in both locations in this file?

Reply:
I'd omit it on both places...

Reply:
OK